server/src/main/java/org/elasticsearch/search/fetch/subphase/FetchSourceContext.java

```diff
@@ -133,6 +206,7 @@ public static FetchSourceContext fromXContent(XContentParser parser) throws IOEx
         boolean fetchSource = true;
         String[] includes = Strings.EMPTY_ARRAY;
         String[] excludes = Strings.EMPTY_ARRAY;
+        boolean includeVectors = Boolean.TRUE;
```

Comment:
As far as I understand, we want to set the default `includeVectors` value to `null` in this path.